boards/v2/google/google_kukui/Kconfig.google_kukui

```diff
@@ -0,0 +1,6 @@
+# Copyright 2019 The Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_GOOGLE_KUKUI
+	select SOC_SERIES_STM32F0X
```

Comment:
```suggestion
```

Reply:
Damn, these were supposed to be gone with latest scripts changes.
Need to look at the script

Reply:
Hopefully now fixed by https://github.com/zephyrproject-rtos/zephyr/pull/68170